modules/sublimeBidAdapter.js

```diff
@@ -9,6 +9,28 @@ const DEFAULT_PROTOCOL = 'https';
 const SUBLIME_VERSION = '0.1';
 let SUBLIME_ZONE = null;
 
+/**
+ * Send a pixel to antenna
+ * @param {String} name The pixel name
+ * @param {viaSublime} viaSublime
+ */
+function sendAntennaPixel(name, viaSublime) {
```

Comment:
`viaSublime` not necessary, it really depends on `(typeof top.sublime !== 'undefined')`